phpunit/functional/AutoloadTest.php

```diff
@@ -81,6 +82,7 @@ public function testAutoloadGlpiEvent()
         $this->assertTrue(class_exists('Glpi\\Event'));
     }
 
+    #[RunInSeparateProcess]
```

Comment:
For this one, I suspect that the plugin is not listed in the `Plugin::$activated_plugins`/`Plugin::$loaded_plugins` variables.

Maybe the test should ensure that the plugin is activated.